modules/ROOT/pages/design-delete-project.adoc

```diff
@@ -0,0 +1,23 @@
+= Delete a Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can delete an API-specification project or an API-fragment project.
```

Comment:
Should we add anything about WHY the audience wants or needs to do this?

Reply:
No. People know when they want to delete a project.